pages/wishlist/wishlist.html

```diff
@@ -0,0 +1,205 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Apex Furniture | Wishlist</title>
+    <link rel="stylesheet" href="/styles.css" />
+  </head>
+  <body>
+    <!-- Header -->
+    <nav class="navbar">
+      <div class="logo font-weight-black align-items-center h2">
+        <a
+          class="display-flex font-weight-black align-items-center h2"
+          href="/index.html"
+        >
+          <span class="site-name">Apex &nbsp;</span><span>Furniture</span>
```

Comment:
everything is fine 

but  span mainly use to --> display out of box design ==> different from all 
using span twice together  may be fix